cumulusci.yml

```
@@ -37,6 +37,27 @@ tasks:
         options:
             path: unpackaged/config/dev_delete
 
+    deploy_encryption_permset:
+        group: 'Shield Platform Encryption'
+        description: Deploys "Manage Encryption Keys" permission set.
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/config/spe
```

```suggestion
            path: unpackaged/config/spe
            namespace_inject: $project_config.project__package__namespace
```

We don't have any tokens yet, but generally a good practice to set the project as default.